Source/RunActivity/Content/SceneryShader.fx

```diff
@@ -641,31 +581,17 @@ float4 PSSignalLight(in VERTEX_OUTPUT In) : COLOR0
 //            and pixel shader versions within each technique/pass.           //
 ////////////////////////////////////////////////////////////////////////////////
 
-technique ImagePS2 {
-	pass Pass_0 {
-		VertexShader = compile vs_4_0_level_9_1 VSGeneral2();
-		PixelShader = compile ps_4_0_level_9_1 PSImage2();
-	}
-}
-
-technique ImagePS3 {
-	pass Pass_0 {
-		VertexShader = compile vs_4_0_level_9_3 VSGeneral3();
-		PixelShader = compile ps_4_0_level_9_3 PSImage3();
-	}
-}
-
-technique TransferPS2 {
+technique ImagePS {
 	pass Pass_0 {
-		VertexShader = compile vs_4_0_level_9_1 VSTransfer2();
-		PixelShader = compile ps_4_0_level_9_1 PSImage4();
+		VertexShader = compile vs_4_0_level_9_3 VSGeneral();
+		PixelShader = compile ps_4_0_level_9_3 PSImage();
```

Comment:
Does bumping the level here affect what hardware is supported?

Reply:
https://en.wikipedia.org/wiki/Feature_levels_in_Direct3D
in short: yes. 
In reality I doubt there is much hardware from the ~before 2004 area around.

Reply:
Probably true 🙂

And since "multiple render targets" is going to be needed 9_3 seems like a good minimum anyway. Do you know if it matters that they're currently a mixture of 9_1 and 9_3? Would it be useful to make every shader 9_3?

Reply:
probably better to avoid different version, I've just updated that.